site/content/docs/5.3/examples/form/form.js

```diff
@@ -16,6 +16,16 @@
         // Focus on first error
         const invalidItems = form.querySelectorAll(':invalid')
         invalidItems[0].focus()
+        // Add the id of the corresponding invalid message to each invalid field
+        invalidItems.forEach(element => {
+          const valuesArray = [`${element.id}Label`, `${element.id}Feedback`].join(' ')
+          element.setAttribute('aria-labelledby', valuesArray)
+        })
+        // Remove the id of the corresponding invalid message to each valid field
+        const validItems = form.querySelectorAll(':valid')
+        validItems.forEach(element => {
+          element.setAttribute('aria-labelledby', `${element.id}Label`)
```

Comment:
Same here but with valid feedback ? Or did we say that there won't be any valid feedback in Boosted ?

Reply:
No need to do it for valid feedback. In Boosted a valid field can be _implicytly_ stated as valid because of not being _explicitly_ stated as invalid.